### ISSUE TITLE
staking probability 

### ISSUE THREAD

**Issue by stoddy15** [ThePharros/probability-calculator#1]
can you please add the probability of winning your current stake?

in the staking community we all have max combat

would be interesting to see

**Comment by ThePharros** [comment]
I don’t know how staking works but can look in to it. It would have to assume static factors though as personal skill such as timing attacks and movement would vary. 

If everyone is max stats and using the same items and attack at the same tick with the same attack style, what would make the odds not be 50-50?

**Comment by stoddy15** [comment]
Yes the odds should be 50-50 but I don't know if it is. I just know that the sand casino is a huge appeal to Runescape players.

What about pvp and bounty hunter probability shouldn't that be easier to calculate? 

Combat level differs, items worn etc

**Comment by ThePharros** [comment]
A lot of those variables depend on external factors such as player skill/timing, connectivity, etc. There wouldn't be an equation to determine such odds.

As far as PvP goes, I believe there are already combat calculators out there to determine accuracy % and DPS and whatnot, which can get pretty complex tbh.

**Comment by stoddy15** [comment]
I think you may be mistaken. Runescape is a click based game so no skill is involved. When was the last time you logged in and played?

couldn't you calculate their mage level to see if they have the ability to tele block?

please be open minded, it will set you free

**Comment by ThePharros** [comment]
Click precision and timing vary among players, so anything that has those as determining factors wouldn't be deterministic and in fact dependent on person to person skill. Calculating DPS based on gear and lvls, or determining if they can tb? Can definitely be done, but that's not what you asked before. Calculating the probability of an arbitrary person teleing away or when they will eat food is meaningless.

**Comment by stoddy15** [comment]
> Click precision and timing vary among players, so anything that has those as determining factors wouldn't be deterministic and in fact dependent on person to person skill. Calculating DPS based on gear and lvls, or determining if they can tb? Can definitely be done, but that's not what you asked before. Calculating the probability of an arbitrary person teleing away or when they will eat food is meaningless.

apologises I have a hard time getting my ideas across. I have dyslexia 

that is what I mean.

**Comment by ThePharros** [comment]
In that case it wouldn't be probabilistic if you just want to know if they have the stats to do something or not. For things like DPS, there are already tools out there to determine those, and would actually be a lot of work to implement, but may be something I'd be open to down the road. Just not anytime soon, sorry.
